mockintosh/stats.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+.. module:: __init__
+    :synopsis: module that contains server classes.
+"""
+
+
+class BaseStats:
+    def __init__(self):
+        self.parent = None
+        self.children = None
+        self.request_counter = 0
+        self.status_code_distribution = {}
```

Comment:
`collections.Counter()` is a convenient standard class for this task

Reply:
Fixed with https://github.com/up9inc/mockintosh/commit/2933ff59a02f8058dd71662820583ce2888626cb